### PR TITLE
Frontend Changes

### PR DESCRIPTION
**Pull request by Dsenza**
- Move index to root so frontend loads from localhost:8080/
- cleaned out a lot of old, useless, bad, or otherwise unneeded files and frontend code
- create a user meals list, and get it fetching and displaying the currently logged in user's meals.
- Closes Issue #43

### CODE REVIEW

**Comment by wdoug** [comment]
> - Move index to root so frontend loads from localhost:8080/

The static file server for production will need to be updated for this

File: client/src/components/FoodList.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ class FoodList extends React.Component {
   }
 
   componentWillMount() {
-  	this.props.fetchFood(this.props.location.query.userToken);
+    const userToken = localStorage.getItem('userToken');
```
Comment (wdoug) [inline client/src/components/FoodList.js:13]: We'll probably just want to have this as part of the api directly.

**Comment by wdoug** [comment]
After some head scratching I was able to get the client being served at `/` by specifying `contentBase: path.resolve(__dirname, '../../client')` as an option to the WebpackDevServer in `01-frontend-server`.

**Comment by Dsenza** [comment]
I'll set that up tomorrow evening, and move the index.html back to where it was.

Unless you want to push a commit to this branch.

**Comment by wdoug** [comment]
Went ahead and created pr: https://github.com/codefordenver/ketohero/pull/66 into your branch. If all looks well go ahead and merge (preferably with squash).

**Pull request merged**